aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -20,7 +20,396 @@ The model classes you create are known as POCO classes (from **P**lain **O**ld *
 
 In this tutorial, you write the model classes first, and EF Core creates the database.
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
+
+## Add a data model class
+
+# [Visual Studio](#tab/visual-studio)
+
+Right-click the *Models* folder > **Add** > **Class**. Name the file *Movie.cs*.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+Add a file named *Movie.cs* to the *Models* folder.
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+Right-click the *Models* folder > **Add** > **New Class** > **Empty Class**. Name the file *Movie.cs*.
+
+---
+
+Update the *Movie.cs* file with the following code:
+
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Models/Movie.cs)]
+
+The `Movie` class contains an `Id` field, which is required by the database for the primary key.
+
+The <xref:System.ComponentModel.DataAnnotations.DataType> attribute on `ReleaseDate` specifies the type of the data (`Date`). With this attribute:
+
+* The user is not required to enter time information in the date field.
+* Only the date is displayed, not time information.
+
+[DataAnnotations](/dotnet/api/system.componentmodel.dataannotations) are covered in a later tutorial.
+
+## Add NuGet packages
+
+# [Visual Studio](#tab/visual-studio)
+
+From the **Tools** menu, select **NuGet Package Manager** > **Package Manager Console** (PMC).
+
+![PMC menu](~/tutorials/first-mvc-app/adding-model/_static/pmc.png)
+
+In the PMC, run the following command:
+
+```powershell
+Install-Package Microsoft.EntityFrameworkCore.SqlServer
+```
+
+The preceding command adds the EF Core SQL Server provider. The provider package installs the EF Core package as a dependency. Additional packages are installed automatically in the scaffolding step later in the tutorial.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/add-EF-NuGet-SQLite-CLI.md)]
+
+<!-- [!INCLUDE[](~/includes/add-EF-NuGet-SQLite-CLI-5.md)]
```

Comment:
As soon as this merges to master, fix the preceding lines for the right version.